boards/nxp/frdm_mcxa156/board.c

```diff
@@ -184,6 +184,12 @@ void board_early_init_hook(void)
 	SPC_EnableActiveModeAnalogModules(SPC0, (kSPC_controlCmp0 | kSPC_controlCmp0Dac));
 #endif
 
+#if DT_NODE_HAS_STATUS_OKAY(DT_NODELABEL(opamp0))
+	RESET_ReleasePeripheralReset(kOPAMP0_RST_SHIFT_RSTn);
+	CLOCK_EnableClock(kCLOCK_GateOPAMP0);
+	SPC_EnableActiveModeAnalogModules(SPC0, kSPC_controlOpamp0);
+#endif
+
```

Comment:
I think this is not a portable way to enable a peripheral. Please consider adding all required properties to the dts-binding and make the driver setup clocks, resets and etc. as needed.

Reply:
yes, but this is WIP issue for NXP enablement, we don't have the reset and clock control drivers in place very well, so should not be a requirement for this PR, the technical debt is already there with all other peripherals just about